http/codegen/openapi/openapi_v2_builder.go

```diff
@@ -158,8 +158,8 @@ func securitySpecFromExpr(root *expr.RootExpr) map[string]*SecurityDefinition {
 			sd.Type = "basic"
 		case expr.APIKeyKind:
 			sd.Type = "apiKey"
-			sd.In = s.In
-			sd.Name = s.Name
+			sd.In = "header"
```

Comment:
Why are we hardcoding here? API keys can be set in parameters as well for example (maybe we need a test for that).

Reply:
This defines the top-level `securityDefinitions` object. It is possible that the there are two endpoints secured by API key where one passes the key in the header and the other passes in the param, correct?